packages/odie-client/src/components/notices/use-connection-status-notice.tsx

```diff
@@ -1,52 +1,59 @@
 import { HelpCenterSelect } from '@automattic/data-stores';
 import { HELP_CENTER_STORE } from '@automattic/help-center/src/stores';
-import { Notice } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
+import { Icon, check, warning, info } from '@wordpress/icons';
 import { useI18n } from '@wordpress/react-i18n';
 import { useEffect, useState } from 'react';
 
-export const ConnectionStatus = () => {
+export default function useConnectionStatusNotice( isLiveChat: boolean = false ) {
 	const [ shouldWarn, setShouldWarn ] = useState( false );
 	const { __ } = useI18n();
 
-	const connectionStatusMap = {
-		disconnected: {
-			status: 'error',
-			message: __( 'Chat connection lost.', __i18n_text_domain__ ),
-		},
-		reconnecting: {
-			status: 'warning',
-			message: __( 'Reconnecting…', __i18n_text_domain__ ),
-		},
-		connected: {
-			status: 'success',
-			message: __( 'You’re back online with support.', __i18n_text_domain__ ),
-		},
-	} as const;
-
 	const connectionStatus = useSelect(
 		( select ) => ( select( HELP_CENTER_STORE ) as HelpCenterSelect ).getZendeskConnectionStatus(),
 		[]
 	);
 
 	useEffect( () => {
+		if ( ! isLiveChat ) {
+			setShouldWarn( false );
+			return;
+		}
+
 		if ( connectionStatus === 'disconnected' ) {
 			setShouldWarn( true );
 		}
-	}, [ connectionStatus ] );
+	}, [ connectionStatus, isLiveChat ] );
+
+	if ( ! isLiveChat ) {
+		return undefined;
+	}
```

Comment:
Is this needed? `shouldWarn` is false when `isLiveChat` is true, so the `if ( ! shouldWarn )` check below should catch that scenario 🤔